.jenkins/actions/lint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Does this lint action block any tests? If not, we can remove it and rely on CircleCI for linting, since it gets answers in about 10 seconds. I had thought lint checks occur as part of other actions before the tests are run.

Reply:
This script runs before installing virtual env on daint, so it will block any test runs.